packages/plugins/ansible-collection-theforeman-foreman/ansible-collection-theforeman-foreman.spec

```diff
@@ -25,18 +17,19 @@ Source0:    https://galaxy.ansible.com/download/%{collection_namespace}-%{collec
 BuildArch:  noarch
 
 Provides: ansible-collection(%{collection_namespace}.%{collection_name}) = %{version}
-Provides: bundled(%{ansible_python}-apypie) = 0.3.2
+Provides: bundled(python-apypie) = 0.3.2
```

Comment:
Shouldn't this by python 2 vs 3?

Reply:
or python38? or python39…

technically, the code we vendor is version agnostic, and doesn't live in any python-specific (loadable) path, so no idea